octobot_services/interfaces/util/portfolio.py

```diff
@@ -51,20 +54,24 @@ def get_portfolio_current_value():
     has_simulated_trader = False
 
     for exchange_manager in interfaces.get_exchange_managers():
-        if trading_api.is_trader_enabled(exchange_manager):
+        try:
+            if trading_api.is_trader_enabled(exchange_manager):
 
-            current_value = trading_api.get_current_portfolio_value(exchange_manager)
+                current_value = trading_api.get_current_portfolio_value(exchange_manager)
 
-            # current_value might be 0 if no trades have been made / canceled => use origin value
-            if current_value == 0:
-                current_value = trading_api.get_origin_portfolio_value(exchange_manager)
+                # current_value might be 0 if no trades have been made / canceled => use origin value
+                if current_value == 0:
+                    current_value = trading_api.get_origin_portfolio_value(exchange_manager)
 
-            if trading_api.is_trader_simulated(exchange_manager):
-                simulated_value += current_value
-                has_simulated_trader = True
-            else:
-                real_value += current_value
-                has_real_trader = True
+                if trading_api.is_trader_simulated(exchange_manager):
+                    simulated_value += current_value
+                    has_simulated_trader = True
+                else:
+                    real_value += current_value
+                    has_real_trader = True
+        except AttributeError:
+            # no activated trader
```

Comment:
It's now possible to have no activated trader ?

Reply:
Yes, to disable any kind of trading and no only backtestings for example